Documentation Update: Add linux ssh guide (#1527)
LizardByte/Sunshine: docs/source/about/usage.rst

      .. Attention:: The configuration file specified will be created if it doesn't exist.

   **Start Sunshine over SSH (Linux/X11)**
      Assuming you are already logged into the host, you can use this command

      .. code-block:: bash

         ssh <user>@<ip_address> 'export DISPLAY=:0; sunshine'
      

>       If you are logged into the host with only a tty, you can use ``startx`` to start the X server prior to executing sunshine. You nay need to add ``sleep`` between ``startx`` and ``sunshine`` to allow more time for the display to be ready.

Reviewer: ```suggestion
      If you are logged into the host with only a tty (teletypewriter), you can use ``startx`` to start the X server prior to executing sunshine.
      You nay need to add ``sleep`` between ``startx`` and ``sunshine`` to allow more time for the display to be ready.
```

Just split somewhere to keep under 120 characters.

      .. code-block:: bash

         ssh <user>@<ip_address> 'startx &; export DISPLAY=:0; sunshine'
      

>       .. Note:: You could also utilize the ``~/.bash_profile`` or ``~/.bashrc`` files to setup the ``DISPLAY``

Reviewer: ```suggestion
      .. tip:: You could also utilize the ``~/.bash_profile`` or ``~/.bashrc`` files to setup the ``DISPLAY``
```

Let's change this to `tip`.

         variable.
      

>       .. admonition:: Guide
>         :class: important
> 
>         See :ref:`remote-ssh-headless-setup` on how to setup a headless streaming server without autologin and dummy plugs (X11 + NVidia GPUs)

Reviewer: ```suggestion
      .. seealso::

         See :ref:`remote-ssh-headless-setup` on how to setup a headless streaming server without autologin and dummy plugs (X11 + NVidia GPUs)
```

- Older games that use NVIDIA-specific NVAPI HDR rather than native Windows 10 OS HDR support may not display in HDR.
- Some GPUs can produce lower image quality or encoding performance when streaming in HDR compared to SDR.

> Tutorials and Guides
> --------------------
> Tutorial videos are available `here <https://www.youtube.com/playlist?list=PLMYr5_xSeuXAbhxYHz86hA1eCDugoxXY0>`_.  
> Guides are available :doc:`here <./guides/guides>`

Reviewer: ```suggestion
Tutorials and Guides
--------------------
Tutorial videos are available `here <https://www.youtube.com/playlist?list=PLMYr5_xSeuXAbhxYHz86hA1eCDugoxXY0>`_.

Guides are available :doc:`here <./guides/guides>`.
```
